src/main.rs

```diff
@@ -1,37 +1,36 @@
 mod cel;
-mod light;
-mod outline;
 mod color;
-mod scale;
 mod geometry;
+mod light;
+mod loaders;
 mod material;
+mod outline;
+mod scale;
 
 use std::f32::consts::PI;
 use std::time::Duration;
-use std::path::Path;
 use std::thread;
-use std::rc::Rc;
 
 use euc::{Pipeline, rasterizer, buffer::Buffer2d, Target};
 use minifb::{self, Key, KeyRepeat};
-use tobj;
 use vek::{Mat4, Vec2, Vec3, Vec4, Rgba};
 use image::ImageBuffer;
 
 use crate::cel::CelShader;
-use crate::outline::OutlineShader;
+use crate::color::{rgba_to_bgra_u32, bgra_u32_to_rgba, vek_rgba_to_image_rgba};
+use crate::geometry::Mesh;
 use crate::light::DiffuseLight;
+use crate::loaders::*;
+use crate::outline::OutlineShader;
 use crate::scale::scale_buffer;
-use crate::geometry::Mesh;
-use crate::material::Material;
-use crate::color::{rgba_to_bgra_u32, bgra_u32_to_rgba, vek_rgba_to_image_rgba};
 
 fn main() {
     let image_width = 64;
     let image_height = 64;
 
+    // let frames = [GltfLoader::load_file("samples/bigboi/gltf/bigboi_rigged.gltf")];
```

Comment:
You can delete this commented out line since it was probably included accidentally